pytest_selenium/drivers/firefox.py

```diff
@@ -31,12 +31,14 @@ def pytest_addoption(parser):
                      help='path to a firefox extension.')
 
 
-def driver_kwargs(capabilities, driver_path, firefox_options, **kwargs):
+def driver_kwargs(capabilities, driver_path, firefox_options, log_path,
+                  pytestconfig, **kwargs):
     kwargs = {}
     if capabilities:
         kwargs['capabilities'] = capabilities
     if driver_path is not None:
         kwargs['executable_path'] = driver_path
+    pytestconfig._driver_log = log_path
```

Comment:
Where is `log_path` implemented? We also need to pass the path to the log as a keyword argument. This means adding something like:

```python
log_path = os.path.realpath('geckodriver.log')
kwargs['log_path'] = pytestconfig._driver_log = log_path
```